app/javascript/pages/Browse.vue

```diff
@@ -18,7 +18,7 @@
       <section>
         <BrowserSelector :browser="browser" @clicked="browser = $event" />
       </section>
-      <component :is="browser" :contributions="activeContributions" class="row" />
+      <component :is="browser" :contributions="activeContributions" class="row" accessToken="pk.eyJ1IjoibXV0dWFsLWFpZC1hcHAiLCJhIjoiY2tmZTBvd3UwMDBhbTJ4cDlic2JmMWZoaiJ9.rWscBjdl1SMT5N0yekIJYg"/>
```

Comment:
A series of questions:

- Is `accesToken` a credential to the maps API? 
- Is this a sensitive credential? Does this get served down to the client? Is there any amount of armoring necessary to protect it?

Reply:
This also seems to be copy-pasta'd in `pp/javascript/pages/browse/MapBrowser.vue`. Channeling Dave Thomas and Andy Hunt: "There should exist a single authoritative source of knowledge within a system - IE Do not repeat yourself."

Had you considered the case where we need to change the Map API credential? It seems likely that we might miss one or more instances with the current design.

Reply:
@indiebrain yup, i believe we agreed to go with this path for this week, but that there's a ticket to move this out asap